aea/cli/add.py

```diff
@@ -56,8 +57,18 @@
 def add(click_context: click.Context, local: bool, remote: bool) -> None:
     """Add a package to the agent."""
     ctx = cast(Context, click_context.obj)
+    enforce(
+        not (local and remote), "'local' and 'remote' options are mutually exclusive."
+    )
+    if not local and not remote:
+        try:
+            ctx.registry_path
+        except ValueError as e:
+            click.echo(f"{e}\nTrying remote registry (`--remote`).")
+            remote = True
```

Comment:
When the registry path is not specified then we swap to remote. But we tell the user about this!

Currently, code added in a non-DRY fashion to avoid too many changes in code base. We should refactor the CLI for v2 to improve DRYness.